build_scripts/build_usd.py

```diff
@@ -311,7 +320,7 @@ def FormatMultiProcs(numJobs, generator):
     tag = "-j"
     if generator:
         if "Visual Studio" in generator:
-            tag = "/M:"
+            tag = "/M:" # This will build multiple projects at once. Also add /MP to CXX_FLAGS (below).
```

Comment:
I did a quick test today on a Windows VM running MSVC 2015 with 24 virtual CPUs and got some surprising results:

USD build with `/M`: ~10 minutes
USD build with `/p:CL_MPcount=`: ~25 minutes (!)

I verified that all cores were being used during the build, so I don't know what's going on here. Did you notice a performance improvement on your side with your original change?

It might be worth pulling this out into a separate PR so we can dig into it further without holding up the rest of this change.